src/sentry/static/sentry/app/views/substances/substances.jsx

```diff
@@ -92,7 +93,8 @@ class Substances extends React.Component {
       {
         Header: 'Volume',
         id: 'volume',
-        accessor: d => (d.properties.volume ? d.properties.volume.value : null),
+        accessor: d =>
```

Comment:
There is support for (localized) number formatting in the `Intl.NumberFormat` object which you might want to check out. It's supported in all relevant browsers:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat
https://caniuse.com/#search=DateTimeFormat